rendermodules/em_montage_qc/plots.py

```diff
@@ -0,0 +1,90 @@
+import numpy as np
+import matplotlib.pyplot as plt
```

Comment:
I think these matplotlib imports are not being used.

Reply:
Commenting the matplotlib imports... will be needing them in future for point match qc plots at the montage level.